static/js/components/dashboard/CoursePrice.js

```diff
@@ -11,25 +16,79 @@ import { formatPrice } from '../../util/util';
 
 export default class CoursePrice extends React.Component {
   props: {
-    course: Course,
+    course: Course
   };
 
+  renderTooltip(text: string): React$Element<*>|void {
+    let tooltip;
+
+    if (text) {
```

Comment:
i think you should just return the `<div>` directly in this function and not worry about whether the `text` value is blank